mc/Diameter.package/AVPCollection.class/instance/^slash.st

```diff
@@ -0,0 +1,6 @@
+accessing
+/ aName
+	^ elements
+		detect: [ :avp | avp avpData avpType avpName = path ]
+		ifFound: [ :element | element avpData / aName ]
+		ifNone: [ nil ]
```

Comment:
This may trigger lots of DNU? Can we return something empty here? Does it make sense to you?

Reply:
@zecke good point! I'd try to give preference to the consumer
The alternatives I think we have are:
1. use `select:` (instead of `detect:`), hence returning always a collection, empty on no matches
2. return `nil` on no matches, giving preference to `nil` returns as is (potentially DNUing as you pointed out)
3. raising an exception on no-match

What do you think is best?  cc/ @ranomine

Reply:
It is probably desirable to differentiate between empty value and no value? WDUT about adding And AVPEmptyCollection and something a long the lines of >>#ifHasValue: [:val | ] >>#ifNotHasValue: []?

Reply:
I've made it respond an empty `AVPCollection` which seems enough so far